### PR TITLE
Rework of the Belgian Question & First content for William I

### PR DESCRIPTION
**Pull request by GeneralVonGelre**
Pru-Net Alliance still in the works.

William I still requires more flavour

All localisation should be accounted for.

### CODE REVIEW

File: TGC/common/event_modifiers.txt [hunk 1]
```diff
@@ -7883,6 +7883,11 @@ Backed-Down-Manchuria = {
 
 ###End of German Extra Flavor ###
 
+###Netherlands Flavor###
+willem_i_absolutism = {}
```
Comment (anzu-3278) [inline TGC/common/event_modifiers.txt:7594]: You sure you don't need an icon here?

File: TGC/decisions/FranceExtra.txt [hunk 2]
```diff
@@ -568,4 +568,44 @@ political_decisions = {
...
+			NOT = {
+				relation = { who = ENG value = 0 }
+				relation = { who = PRU value = 0 }
+			 }
+		}
+
+		effect = {
+			ENG = { country_event = 36719 }
+			badboy = 3
+			FRA = { set_country_flag = talleyrand_proposed }
```
Comment (anzu-3278) [inline TGC/decisions/FranceExtra.txt]: Not sure how sending events to nonexistent countries works, maybe just do random_country instead?

File: TGC/common/event_modifiers.txt (hunk 1 above)
Comment (GeneralVonGelre) [reply TGC/common/event_modifiers.txt:7594]: Not really no, in the future there will be some effect to them as well, but I think the red hammer it defaults to works just fine

File: TGC/decisions/FranceExtra.txt (hunk 2 above)
Comment (GeneralVonGelre) [reply TGC/decisions/FranceExtra.txt]: in my tests it didn't seem to break anything, but sure I can change it to a random_county effect

**Pull request closed without merging**